accessibility/README.md

```diff
@@ -0,0 +1,37 @@
+# Accessibility
+
+Accessibility is the practice of making the tools and websites we build usable by everyone — however they encounter it. As designers of websites, we should try to accommodate all potential users in as many contexts of use as we can. Doing so has broader benefits — notably better designs for all. 
+
+Accessibility is a growing practice at DataMade, and while many of our sites follow web best practices, explicitly testing for accessibility is only something we've started to undertake in 2021 and 2022.
+
+## Beware of 'overlay' tools that 'solve' accessibility
+
+There are a number of tools, like UserWay, that you can purchase that act as plugins to sit on top of your site and provide accessibility tools. However, accessibility advocates warn against using these tools:
+
+> To laypersons, these features may seem beneficial, but their practical value is largely overstated because the end users that these features claim to serve will already have the necessary features on their computer, either as a built-in feature or as an additional piece of software that the user needs to access not only the Web but all software.
+> 
+> On this latter point, it is a mistake to believe that the features provided by the overlay widget will be of much use by end users because if those features were necessary to use the website, they'd be needed for all websites that the user interacts with. Instead, the widget is  —at best—redundant functionality with what the user already has.
+
+For more on this, see https://overlayfactsheet.com
+
+## Key accessibility considerations
+
+The basic approach involves ensuring that the HTML and layout of our websites is structured properly, and that visual elements have `alt` tags or descriptions:  
```

Comment:
should we link to [the mdn docs about accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility)? this is my go-to resource

Reply:
also added